doc/md/privacy.md

```diff
@@ -469,51 +473,54 @@ func DenyMismatchedTenants() privacy.MutationRule {
 		}
 		// Query the tenant-id of all users. Expect to have exact 1 result,
 		// and it matches the tenant-id of the group above.
-		uid, err := m.Client().User.Query().Where(user.IDIn(users...)).QueryTenant().OnlyID(ctx)
+		id, err := m.Client().User.Query().Where(user.IDIn(users...)).QueryTenant().OnlyID(ctx)
 		if err != nil {
-			return privacy.Denyf("querying the tenant-id %w", err)
+			return privacy.Denyf("querying the tenant-id %v", err)
 		}
-		if uid != tid {
-			return privacy.Denyf("mismatch tenant-ids for group/users %d != %d", tid, uid)
+		if id != tid {
+			return privacy.Denyf("mismatch tenant-ids for group/users %d != %d", tid, id)
 		}
 		// Skip to the next privacy rule (equivalent to return nil).
 		return privacy.Skip
 	})
-	// Evaluate the mutation rule only on group creation.
-	return privacy.OnMutationOperation(rule, ent.OpCreate)
 }
 ```
 
 We add this rule to the `Group` schema and run code-generation.
 
-```go
+```go title="examples/privacytenant/ent/schema/group.go"
 // Policy defines the privacy policy of the Group.
 func (Group) Policy() ent.Policy {
 	return privacy.Policy{
 		Mutation: privacy.MutationPolicy{
-			rule.DenyMismatchedTenants(),
+			// Limit DenyMismatchedTenants only for
+			// Create operation
+			privacy.OnMutationOperation(
+				rule.DenyMismatchedTenants(),
+				ent.OpCreate,
+			),
 		},
 	}
 }
 ```
 
 Again, we expect the privacy-rules to take effect on the client operations.
 
-```go
+```go title="examples/privacytenant/example_test.go"
 func Do(ctx context.Context, client *ent.Client) error {
     // A continuation of the code-block above.
 
-	// We expect operation to fail, because the DenyMismatchedTenants rule
-	// makes sure the group and the users are connected to the same tenant.
-	_, err = client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(nati).Save(admin)
+	// Expect operation to fail, because the DenyMismatchedTenants rule makes sure
+	// the group and the users are connected to the same tenant.
```

Comment:
```suggestion
	// Expect operation to fail because the DenyMismatchedTenants rule
	// makes sure the group and the users are connected to the same tenant.
```